meta/validation.json

```diff
@@ -68,14 +66,16 @@
             }
         }
     },
+    "default": {
+        "minContains": 1
+    },
```

Comment:
I'm not a fan of this format, and I don't think we should encourage it.  I think the default value should stay with the property declaration.